### ISSUE TITLE
Want to feed the seed recoverer my own set of seeds [Reward]

### ISSUE THREAD

**Issue by Auxiliumdesign** [gurnec/btcrecover#314]
Hi, 

I have accidentally screwed up my order and amount of words in many ways, way to complicated for the typical phases of the seedrecovery.py program. However I have generated all the possible mnemonics already, saved them in a text document.

I wonder if there is a way to feed my seeds into the program, instead of making it generate it's own seeds during the phases. I recon it should be equally fast, or even slightly faster because of no generation process?

The wallet should contain at least 1 bitcoin, so there will be a reward for anyone who could help me resolve this.

**TL;DR 
Feed my own list of seeds into the seed checker**

**Comment by successhalf** [comment]
I can do this
But at least I have to know your messy words

**Comment by jonathancross** [comment]
Were you able to resolve this?
If not:
1. Is there a passphrase in addition to the seed phrase?
2. Do you know the first address (index `0`) of the wallet you are targeting?

**Comment by 3rdIteration** [comment]
I have added the seedlist functionality into my fork of BTCRecover over here https://github.com/3rdIteration/btcrecover

**Comment by Auxiliumdesign** [comment]
Would I be able to have 15 words looking for a 12w seed?

**Comment by 3rdIteration** [comment]
Yea, that will produce about 220 billion combination. Is it a Electrum seed or BIP39 seed? (Electrum seeds are faster to search)

On a home PC it would take a few months to exhaust the search, but a 48 core Linode (or thread ripper) could knock it over in about a month.

**Comment by Auxiliumdesign** [comment]
It is a bip seed i recon, as it is almost 3 years old. Im pretty sure it is
also of the old style adress starting with a 1.

**Comment by 3rdIteration** [comment]
Yea that won't be a problem. You could use either the tokenlist function or feed your pre-generated seedlist.

I just released a YouTube video over in my channel that looks at how to use it for this.

**Comment by jonathancross** [comment]
@3rdIteration -- Nice work!
Happy to see a revived fork of this project.
PS: I have gone ahead and archived my (very, very basic) [segwit wallet recovery tool](https://github.com/jonathancross/segwit-wallet-recovery) and pointed to your fork of this project.

**Comment by Auxiliumdesign** [comment]
Is this tested in the program before creating wallet?

**Comment by 3rdIteration** [comment]
BTCRecover checks potential seeds for a valid checksum, rejecting invalid ones before moving on to deriving keys.

The listkeys argument doesn't do this (yet) as it's mostly for debugging your tokenlist.

**Comment by 3rdIteration** [comment]
> @3rdIteration -- Nice work!
> Happy to see a revived fork of this project.
> PS: I have gone ahead and archived my (very, very basic) [segwit wallet recovery tool](https://github.com/jonathancross/segwit-wallet-recovery) and pointed to your fork of this project.

Yep, it's too good of a tool to let die :)

**Comment by btcexchanging** [comment]
if you haven't recovered your wallet, i can help you recover your wallet.
I have recovered most of the common wallets and recovery phrases for many users
you can reach me here at my telegram account:  @walletrecover

**Comment by viladmir2022** [comment]
hi, if you still have wallet problem. I can recover your lost wallet.
I have powerful servers and excellent software that can find any password.
and my fee is only 15-25% of the recovered fund
here is my telegram id: @viladmir2022  and Gmail: walletsrecoveryservice@gmail.com
